package.json

```diff
@@ -22,39 +22,41 @@
     "browser"
   ],
   "main": "./dist/vis.js",
+  "module": "./dist/vis.esm.js",
```

Comment:
I don't think we need the esm module here. If people want to bundle vis features the should better use the sup-modules directly. They will be better maintained.
Well, it doesn't do harm to have it but if we later want to remove it it will be missed. Better not provide it from the beginning?